DNN Platform/DotNetNuke.Web.Mvc/DnnMvcHandler.cs

```diff
@@ -57,7 +57,9 @@ bool IHttpHandler.IsReusable
 
         void IHttpHandler.ProcessRequest(HttpContext httpContext)
         {
+#pragma warning disable CS0612 // Type or member is obsolete
```

Comment:
I would not override this warning, we should replace it with the new way or remove it or leave the warning in if not possible.